internal/dic/udic.go

```diff
@@ -87,3 +82,13 @@ func NewUserDic(path string) (udic *UserDic, err error) {
 	udic.Index, err = BuildIndexTable(keys)
 	return
 }
+
+// NewUserDic build a user dictionary from a file.
+func NewUserDic(path string) (udic *UserDic, err error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return
+	}
+	defer f.Close()
```

Comment:
__[gometalinter]__
* [errcheck] error return value not checked (defer f.Close())